codegen/src/main/java/software/amazon/awssdk/codegen/model/service/EndpointRuleSetModel.java

```diff
@@ -15,15 +15,16 @@
 
 package software.amazon.awssdk.codegen.model.service;
 
-import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
What's preventing us from removing json-util as a dependency?

Reply:
Nothing! That's the goal 😁